setup.py

```diff
@@ -63,7 +63,7 @@
     "jinja2<3.1.0",
     "fs>=2.0",
     "marshmallow>2.13.0,<3.0.0",  # same upper pin as reana-server
-    "reana-commons[kubernetes] @ git+https://github.com/reanahub/reana-commons.git@0.95.0a4",
+    "reana-commons[kubernetes]>=0.95.0a5,<0.96.0",
```

Comment:
Since this change is not doing any feature or bug fixes worthy of announcement in the release notes, we could use:

```
chore(master): update reana-commons to 0.95.0a5
```

(Alternatively, we could do this as part of some feature branch that needs upgrading r-commons later down the line, but since it's good to get rid of the hard pin, we could commit as chore this time...)